corehq/apps/domain/views/settings.py

```diff
@@ -581,16 +583,32 @@ def post(self, request, *args, **kwargs):
             except ValidationError as e:
                 messages.error(request, _(e.message))
                 return HttpResponseRedirect(request.path_info)
+            uidb64 = kwargs.get('uidb64')
+            uid = urlsafe_base64_decode(uidb64)
+            user = User.objects.get(pk=uid)
+            couch_user = CouchUser.from_django_user(user)
+            clear_login_attempts(couch_user)
+
+            domain = couch_user.domain if couch_user.is_commcare_user() else None
+            log_user_change(by_domain=None, for_domain=domain, couch_user=couch_user, changed_by_user=couch_user,
+                            changed_via=USER_CHANGE_VIA_WEB, change_messages=UserChangeMessage.password_reset(),
+                            by_domain_required_for_log=False, for_domain_required_for_log=False)
+
+            context = {
+                'username': couch_user.raw_username,
+            }
+            email = couch_user.get_email()
+            if not email:
+                email = couch_user.username
```

Comment:
in what circumstance would `get_email` be `None` but couch_user.username contain the correct email?

Reply:
It was to catch mobile workers' emails since WebUsers does the same thing (`return self.email or self.username`) but I just realized the username for mobile workers are all `@<domain>.commcarehq.org` emails which doesn't make sense to send emails to/won't be receiving password reset emails in the first place. Will remove